src/chrome/content/rules/Abs.ch.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+	Nonfunctional hosts in *.abs.ch:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Alternative Bank Switzerland">
+	<target host="abs.ch" />
+	<target host="www.abs.ch" />
+	<target host="bas.ch" />
+	<target host="www.bas.ch" />
+
+	<!-- E-Banking -->
+	<target host="wwwsec.abs.ch" />
+
+	<!-- Redirection domains -->
+	<target host="alternativebankswitzerland.ch" />
+	<target host="www.alternativebankswitzerland.ch" />
+	<target host="banquealternativesuisse.ch" />
+	<target host="www.banquealternativesuisse.ch" />
+	<target host="alternativebankschweiz.ch" />
+	<target host="www.alternativebankschweiz.ch" />
+	<target host="bancaalternativasvizzera.ch" />
```

Comment:
And last but not least `www.bancaalternativasvizzera.ch` ;)

Reply:
Good catch. Fixed